misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/CodeGenerator.java

```diff
@@ -147,15 +148,17 @@ public void generateClient(String executionPath, String definitionPath, String o
      *
      * @param executionPath  Command execution path
      * @param definitionPath Input Open Api Definition file path
+     * @param reldefinitionPath Relative definition path to be used in the generated ballerina code
      * @param serviceName    service name for the generated service
      * @param outPath        Destination file path to save generated source files. If not provided
      *                       {@code definitionPath} will be used as the default destination path
      * @throws IOException               when file operations fail
      * @throws BallerinaOpenApiException when code generator fails
      */
-    public void generateService(String executionPath, String definitionPath, String serviceName, String outPath)
+    public void generateService(String executionPath, String definitionPath,
+                                String reldefinitionPath, String serviceName, String outPath)
             throws IOException, BallerinaOpenApiException {
-        generate(GenType.GEN_SERVICE, executionPath, definitionPath, serviceName, outPath);
+        generate(GenType.GEN_SERVICE, executionPath, definitionPath, reldefinitionPath , serviceName, outPath);
```

Comment:
there is a formatting issue extra space between `reldefinitionPath ,`

Reply:
Fixed with commit c1f65eb